bin/MadGraph5_aMCatNLO/cards/production/13TeV/WPrime_tb/WPrime_tb_run_card.dat

```diff
@@ -67,7 +67,7 @@
  1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
  1        = alpsfact         ! scale factor for QCD emission vx
  F        = chcluster        ! cluster only according to channel diag
- F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 1        = pdfwgt           ! for ickkw=1, perform pdf reweighting
```

Comment:
Why is this "1"? This, by default in CMS, is recommended to be false

Reply:
We expect the pdf uncertainty to be one of our largest sources of error, and without this flag we couldn't get that information.

Reply:
this has nothing to do with pdf systematics. this is a pdf reweighting in the context of isr. please turn off.

Reply:
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/runcmsgrid_LO.sh#L192
this is the line running systematics. all flags are automatically set so it gets always added.

Reply:
I looked into this and it appears you are right. I have changed the flag back to F